script/Ciel/Day 2/02_01_CIEL02_1_1.txt

```diff
@@ -41,8 +41,7 @@ I am sure that if I suggest this to the head of the family, she would accept it
 [sha:466421a93d9324127b2656b98288526c6cfa45a8]{
 -- Page 6, Offset 3632.
 -- 学校の食堂に問い合わせて、その月の献立を把握した上で、何を食べるのかを決めさせていただきますね？』
-After that, I shall contact the school cafeteria, establish what the menu will be for the month,
- and then make a determination as to what you will be served. Is that acceptable?"
+After that, I shall contact the school cafeteria, find out what's on the menu for the month, and then determine what you will be served. Is that acceptable?"
```

Comment:
Isnt the spacing here should be followed?

Reply:
spacing?